otter/test/worker/test_launch_server_v1.py

```diff
@@ -2423,6 +2421,9 @@ def test_delete_and_verify_limits(self):
         :func:`delete_and_verify` limits the number of delete server requests
         to 1. It also delays response by 1 second
         """
+        set_config_data({"worker": {"delete_server_limit": 1}})
+        self.addCleanup(set_config_data, None)
```

Comment:
I just remembered. I actually started on a generic function that also preserves existing config 1-2 months but hit some roadblock. Let me try to revisit it and see.